redwoods-insurance-salesforce/force-app/main/default/lwc/incidentImageCarousel/incidentImageCarousel.js

```diff
@@ -1,17 +1,10 @@
 import { LightningElement, wire, api } from 'lwc';
-import { getRecord } from 'lightning/uiRecordApi';
 import getRelatedPictures from '@salesforce/apex/IncidentController.findRelatedFiles';
 
 export default class IncidentImageCarousel extends LightningElement {
     @api recordId;
     urls;
     pictures;
```

Comment:
What is `pictures` used for`? Cannot find it in the template.

Reply:
It has been removed.